distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -551,140 +397,34 @@ is_unresolved_symbol value = @Builtin_Method "Meta.is_unresolved_symbol"
-
-       Arguments:
-       - fn: The callback function accepting UUID.
-    on_enter self (fn : Text -> Any | Nothing) =
-        new = instrumentor_builtin "onEnter" [ self.impl, fn ]
-        Instrumentor.Value new
-
-    ## PRIVATE
-       ADVANCED
-
-       Registers callback to be executed when a node/expression evaluation
-       is over. The callback `fn` gets UUID and the computed value (or value
-       of `expression` if specified). Usually
-       the value is _cached_ and returned from `on_enter` callback next time
-       the same expression is evaluated.
-
-       > Example
-         Specify `expression` to _"inline evaluate"_ it.
-            see_a_b uuid:Text ~result =
-              if uuid == "expected-uuid" then
-                IO.println "evalutated to "+result.to_text
+       Value associated with a UUID if any
 
-            Meta.meta .fn . instrument . on_return fn=see_a_b expression="a+b" . activate
-
-       Arguments:
-       - fn: The callback function accepting UUID and computed value
-       - expression: Expression to evaluate on_return - by default Nothing -
-         e.g. to provide the return value of the function
-    on_return self (fn : Text -> Any -> Nothing) expression:(Text | Nothing)=Nothing =
-        new = instrumentor_builtin "onReturn" [ self.impl, fn, expression ]
-        Instrumentor.Value new
-
-    ## PRIVATE
-       ADVANCED
-
-       Registers callback to be executed when a node/expression representing function is about to be called.
-       The callback `fn` shall accept three arguments. The UUID to identify the expression, the function to be
-       invoked and the arguments to pass to the function. The callback can return `Nothing`
-       (in such case the function gets executed with provided arguments) or some other value,
-       which is then returned instead of calling the function.
-
-       Arguments:
-       - fn: The callback function accepting UUID and function value
-    on_call self (fn : Text -> Function -> Vector Any -> Any | Nothing) =
-        new = instrumentor_builtin "onCall" [ self.impl, fn ]
-        Instrumentor.Value new
-
-    ## PRIVATE
-       ADVANCED
-
-       Activates configured instrumentor. Returns managed resource to
-       deactivate the instrumentor later.
-
-       Arguments:
-       - value: The value of the atom in the meta representation.
-    activate self =
-        finalize_instrumentor impl = instrumentor_builtin "deactivate" [ impl ]
-        create action = (instrumentor_builtin action [ self.impl, finalize_instrumentor ]) : Managed_Resource
-        create "activate"
-
-## PRIVATE
-   Returns type specified by fully qualified name.
-
-   Arguments:
-   - fqn: fully qualified name.
-find_type_by_qualified_name : Text -> Any
-find_type_by_qualified_name fqn = @Builtin_Method "Meta.find_type_by_qualified_name"
+    uuid id:Text -> Any = Meta_Helpers.instrumentor_builtin "uuid" id
```

Comment:
I'd like to remove `type Instrumentor` from `Meta` completely, but this `uuid` method is used from here:

https://github.com/enso-org/enso/blob/fb6f842942d9b7a44393147ff4f8a3c2a3f9b678/distribution/lib/Standard/Visualization/0.0.0-dev/src/Widgets.enso#L22

Where to hide this _unique_ `uuid` functionality?

Reply:
I guess it belongs to `Runtime`: 2f78b17ac72d85ad69be640926059037b9690ba0